lib/matplotlib/animation.py

```diff
@@ -636,6 +638,9 @@ def _args(self):
             args += ['-loglevel', 'error']
         return [self.bin_path(), *args, *self.output_args]
 
+# Workaround: Set class docstring from __init__ for Sphinx visibility
+FFMpegFileWriter.__doc__ = inspect.getdoc(FFMpegFileWriter.__init__)
```

Comment:
This doesn't look right; it's replacing the docstring that's already there.

Reply:
This actually works, because `inspect.getdoc` can resolve this, only Sphinx cannot. See https://github.com/matplotlib/matplotlib/issues/22831#issuecomment-3019698791. The proper fix would be to report this upstream to Sphinx, but I don’t have the time right now.

Reply:
To be more specific, @timhoffm suggested setting `FFMpegFileWriter.__init__.__doc__`, but you are setting `FFMpegFileWriter.__doc__`.